lib/editor/selectors/index.js

```diff
@@ -360,10 +381,10 @@ function getControlPointsFromShapePoints (shapePoints: Array<ShapePoint>) {
 function addPatternStopsToShapePoints (
   oldShapePoints: Array<ShapePoint>,
   patternStops: Array<PatternStop>,
-  stops: Array<any>, // TODO: use more exact type
+  activePatternStops: Array<GtfsStop>,
```

Comment:
Would be fantastic to have a unit test for this function too.  Of course it'd be awesome to have unit tests for each possible scenario, but even a test that sent the oldShapePoints variable with a length being less than 2 is a good starting point as this big function has 0% coverage.